circle.yml

```diff
@@ -0,0 +1,43 @@
+machine:
+  services:
+    - docker
+
+dependencies:
+  cache_directories:
+    - "~/docker"
+  override:
+    - docker info
+    - I="image-$(date +%j).tar";if [[ -e ~/docker/$I ]]; then echo "Loading $I";docker load -i ~/docker/$I;fi
+    - >
+        printf '{"commit":"%s","version":"%s","source":"https://github.com/%s/%s","build":"%s"}\n'
+        "$CIRCLE_SHA1"
+        "$CIRCLE_TAG"
+        "$CIRCLE_PROJECT_USERNAME"
+        "$CIRCLE_PROJECT_REPONAME"
+        "$CIRCLE_BUILD_URL"
+        > version.json
+    - cp version.json $CIRCLE_ARTIFACTS
+    - docker build -t app:build .
+    - >
+      I="image-$(date +%j).tar"; mkdir -p ~/docker; rm ~/docker/*; docker save app:build > ~/docker/$I; ls -l ~/docker
```

Comment:
... and then this over-writes the `images-<day-of-year>.tar` file after the new build?